src/pages/Upload.js

```diff
@@ -703,7 +703,7 @@ const DonePanel = ({ workspace, workspace: { workspace: { namespace, name } }, t
     }, [
       p([
         h(Link, {
-          href: Nav.getLink('workspace-data', { namespace: workspace.workspace.namespace, name: workspace.workspace.name }),
+          href: Nav.getLink('workspace-data', { namespace, name }),
```

Comment:
Did the linter catch this specifically, or did it catch something close and you corrected it?

Reply:
The linter caught this, but in the function args - Said that the variables `namespace` and `name` were unused, so i flipped this to use it!